### ISSUE TITLE
Bug | Crash when highlighted text empty

### ISSUE THREAD

**Issue by Owehttamy** [uku3lig/chathighlighter#1]
If you leave the string of text that is meant to be highlighted in the config menu, the game will freeze when attempting to open chat or if there is a chat message already on screen.

**Comment by uku3lig** [comment]
could you please send me the crash report file

**Comment by Owehttamy** [comment]
> could you please send me the crash report file

I phrased it badly, it causes the game to freeze instead of crash, so it doesn't create a crash report. Best I can do is a latest.log, but it doesn't really provide anything. Only mods in this are Fabric API, ukuLib, ChatHighlighter, and BorderlessMining (runs game in borderless windowed mode instead of fullscreen. Never has issues with other mods.) so it's not due to a collision of another mod.

```
[21:01:52] [main/INFO]: Loading Minecraft 1.19.3 with Fabric Loader 0.14.12
[21:01:52] [main/INFO]: Loading 57 mods:
	- borderlessmining 1.1.6+1.19.3
	- chathighlighter 1.0.1+mc1.19.3
	- com_moandjiezana_toml_toml4j 0.7.2 via ukulib
	- fabric-api 0.71.0+1.19.3
	- fabric-api-base 0.4.20+afca2f3e85 via fabric-api
	- fabric-api-lookup-api-v1 1.6.19+49abcf7e85 via fabric-api
	- fabric-biome-api-v1 12.1.0+b5d379b085 via fabric-api
	- fabric-block-api-v1 1.0.4+e022e5d185 via fabric-api
	- fabric-blockrenderlayer-v1 1.1.29+c6af733c85 via fabric-api
	- fabric-client-tags-api-v1 1.0.10+99f9db8085 via fabric-api
	- fabric-command-api-v1 1.2.20+f71b366f85 via fabric-api
	- fabric-command-api-v2 2.1.16+99f9db8085 via fabric-api
	- fabric-commands-v0 0.2.37+df3654b385 via fabric-api
	- fabric-containers-v0 0.1.46+df3654b385 via fabric-api
	- fabric-content-registries-v0 3.5.0+1d37d50285 via fabric-api
	- fabric-convention-tags-v1 1.2.1+8e4e694f85 via fabric-api
	- fabric-crash-report-info-v1 0.2.13+aeb40ebe85 via fabric-api
	- fabric-data-generation-api-v1 11.1.3+87855a7085 via fabric-api
	- fabric-dimensions-v1 2.1.40+48349a3f85 via fabric-api
	- fabric-entity-events-v1 1.5.6+b83334a085 via fabric-api
	- fabric-events-interaction-v0 0.4.39+3baeb27a85 via fabric-api
	- fabric-events-lifecycle-v0 0.2.41+df3654b385 via fabric-api
	- fabric-game-rule-api-v1 1.0.29+99f9db8085 via fabric-api
	- fabric-item-api-v1 2.1.7+e498f5f085 via fabric-api
	- fabric-item-group-api-v1 2.1.10+0a2a0d0b85 via fabric-api
	- fabric-key-binding-api-v1 1.0.30+bc01e09785 via fabric-api
	- fabric-keybindings-v0 0.2.28+df3654b385 via fabric-api
	- fabric-lifecycle-events-v1 2.2.9+23a79c8a85 via fabric-api
	- fabric-loot-api-v2 1.1.19+75e9821185 via fabric-api
	- fabric-loot-tables-v1 1.1.23+9e7660c685 via fabric-api
	- fabric-message-api-v1 5.0.13+6ede1da985 via fabric-api
	- fabric-mining-level-api-v1 2.1.30+49abcf7e85 via fabric-api
	- fabric-models-v0 0.3.26+11ba9c3b85 via fabric-api
	- fabric-networking-api-v1 1.2.16+4017a8cb85 via fabric-api
	- fabric-networking-v0 0.3.33+df3654b385 via fabric-api
	- fabric-object-builder-api-v1 5.3.0+85f102ee85 via fabric-api
	- fabric-particles-v1 1.0.19+f1e4495b85 via fabric-api
	- fabric-registry-sync-v0 2.0.3+2608564685 via fabric-api
	- fabric-renderer-api-v1 2.2.0+9f179aa185 via fabric-api
	- fabric-renderer-indigo 0.7.0+9f179aa185 via fabric-api
	- fabric-renderer-registries-v1 3.2.29+df3654b385 via fabric-api
	- fabric-rendering-data-attachment-v1 0.3.24+afca2f3e85 via fabric-api
	- fabric-rendering-fluids-v1 3.0.17+f1e4495b85 via fabric-api
	- fabric-rendering-v0 1.1.32+df3654b385 via fabric-api
	- fabric-rendering-v1 1.12.0+6dc0edd785 via fabric-api
	- fabric-resource-conditions-api-v1 2.2.1+280be3ab85 via fabric-api
	- fabric-resource-loader-v0 0.10.4+a91e48b785 via fabric-api
	- fabric-screen-api-v1 1.0.40+f1e4495b85 via fabric-api
	- fabric-screen-handler-api-v1 1.3.12+99f9db8085 via fabric-api
	- fabric-sound-api-v1 1.0.7+75e9821185 via fabric-api
	- fabric-transfer-api-v1 2.1.13+f1e4495b85 via fabric-api
	- fabric-transitive-access-wideners-v1 2.3.0+5b9a588b85 via fabric-api
	- fabricloader 0.14.12
	- gs_mclo_java_mclogs-java 2.1.1 via ukulib
	- java 17
	- minecraft 1.19.3
	- ukulib 0.6.1+1.19.3
[21:01:52] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/Matthew/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[21:01:53] [main/INFO]: Compatibility level set to JAVA_17
[21:01:54] [main/WARN]: Invalid registry value type detected for PerfOS counters. Should be REG_DWORD. Ignoring: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PerfOS\Performance\Disable Performance Counters.
[21:01:57] [main/INFO]: Building optimized datafixer
[21:02:02] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[21:02:02] [Render thread/INFO]: Setting user: Owehttamy
[21:02:03] [Render thread/WARN]: A corrupted configuration file was found, overwriting it with the default config
java.lang.RuntimeException: java.lang.IllegalStateException: Invalid value on line 1: text = (?i)(^|\b)(?!<Owehttamy>\s)(Owehttamy|Oweh|Owe)('s|'|\b|\B)
	at com.moandjiezana.toml.Toml.read(Toml.java:74) ~[com_moandjiezana_toml_toml4j-0.7.2-9bb2686e7f16b32f.jar:?]
	at net.uku3lig.ukulib.config.serialization.DefaultConfigSerializer.deserialize(DefaultConfigSerializer.java:56) ~[%5B1.19.3%5D%20UkuLib-0.6.1.jar:?]
	at net.uku3lig.ukulib.config.serialization.DefaultConfigSerializer.deserialize(DefaultConfigSerializer.java:18) ~[%5B1.19.3%5D%20UkuLib-0.6.1.jar:?]
	at net.uku3lig.ukulib.config.ConfigManager.<init>(ConfigManager.java:41) ~[%5B1.19.3%5D%20UkuLib-0.6.1.jar:?]
	at net.uku3lig.ukulib.config.ConfigManager.create(ConfigManager.java:56) ~[%5B1.19.3%5D%20UkuLib-0.6.1.jar:?]
	at net.uku3lig.chathighlighter.ChatHighlighter.<clinit>(ChatHighlighter.java:13) ~[%5B1.19.3%5D%20ChatHighlighter-1.0.1.jar:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53) ~[fabric-loader-0.14.12.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:465) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:205) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.12.jar:?]
Caused by: java.lang.IllegalStateException: Invalid value on line 1: text = (?i)(^|\b)(?!<Owehttamy>\s)(Owehttamy|Oweh|Owe)('s|'|\b|\B)
	at com.moandjiezana.toml.Toml.read(Toml.java:140) ~[com_moandjiezana_toml_toml4j-0.7.2-9bb2686e7f16b32f.jar:?]
	at com.moandjiezana.toml.Toml.read(Toml.java:107) ~[com_moandjiezana_toml_toml4j-0.7.2-9bb2686e7f16b32f.jar:?]
	at com.moandjiezana.toml.Toml.read(Toml.java:72) ~[com_moandjiezana_toml_toml4j-0.7.2-9bb2686e7f16b32f.jar:?]
	... 19 more
[21:02:03] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[21:02:04] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[21:02:05] [Render thread/INFO]: Found non-pack entry 'C:\Users\Matthew\AppData\Roaming\.minecraft\resourcepacks\### Archee's Texture Packs', ignoring
[21:02:05] [Render thread/INFO]: Found non-pack entry 'C:\Users\Matthew\AppData\Roaming\.minecraft\resourcepacks\### Out of date', ignoring
[21:02:05] [Render thread/WARN]: Missing metadata in pack file/LO Texture Pack-20221220T232515Z-001.zip
[21:02:05] [Render thread/WARN]: Removed resource pack minecraft:supporteatinganimation from options because it doesn't seem to exist anymore
[21:02:07] [Render thread/INFO]: Narrator library for x64 successfully loaded
[21:02:07] [Render thread/INFO]: Reloading ResourceManager: vanilla, file/[1] 1.19.3 VanillaTweaks.zip, file/[2] FreshAnimations_v1.7_S9.zip, file/[3] EBE_v1.1_1.19.x.zip, file/[4] §2§lVisual§0-§a§lLeaf§0-§2§lDecay§0-§7§o1.3§0-§8.zip, file/[5] §6§lVisual§0-§e§lBee§0-§6§lItems§0-§7§o1.0§0-§8, file/[6] 1.0.1v_VisualGoatHorns, file/[7] 3D-Breaking.zip, file/[8] Capitalisation-Fixes-v2.16.zip, Fabric Mods (Borderless Mining, ChatHighlighter, Fabric API, Fabric API Base, Fabric API Lookup API (v1), Fabric Biome API (v1), Fabric Block API (v1), Fabric BlockRenderLayer Registration (v1), Fabric Client Tags, Fabric Command API (v1), Fabric Command API (v2), Fabric Commands (v0), Fabric Containers (v0), Fabric Content Registries (v0), Fabric Convention Tags, Fabric Crash Report Info (v1), Fabric Data Generation API (v1), Fabric Dimensions API (v1), Fabric Entity Events (v1), Fabric Events Interaction (v0), Fabric Events Lifecycle (v0), Fabric Game Rule API (v1), Fabric Item API (v1), Fabric Item Group API (v1), Fabric Key Binding API (v1), Fabric Key Bindings (v0), Fabric Lifecycle Events (v1), Fabric Loot API (v2), Fabric Loot Tables (v1), Fabric Message API (v1), Fabric Mining Level API (v1), Fabric Models (v0), Fabric Networking API (v1), Fabric Networking (v0), Fabric Object Builder API (v1), Fabric Particles (v1), Fabric Registry Sync (v0), Fabric Renderer API (v1), Fabric Renderer - Indigo, Fabric Renderer Registries (v1), Fabric Rendering Data Attachment (v1), Fabric Rendering Fluids (v1), Fabric Rendering (v0), Fabric Rendering (v1), Fabric Resource Conditions API (v1), Fabric Resource Loader (v0), Fabric Screen API (v1), Fabric Screen Handler API (v1), Fabric Sound API (v1), Fabric Transfer API (v1), Fabric Transitive Access Wideners (v1), Fabric Loader, ukulib)
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=south,half=top,shape=outer_right,waterlogged=false:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=east,half=bottom,shape=outer_right,waterlogged=true:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=east,half=bottom,shape=outer_left,waterlogged=false:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=east,half=top,shape=outer_left,waterlogged=false:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=south,half=bottom,shape=outer_right,waterlogged=false:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=north,half=top,shape=outer_left,waterlogged=true:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=south,half=top,shape=outer_left,waterlogged=true:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Worker-Main-21/WARN]: Missing textures in model minecraft:stone_stairs#facing=west,half=bottom,shape=outer_right,waterlogged=false:
    minecraft:textures/atlas/blocks.png:minecraft:block/block/variated/stone/15
[21:02:10] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[21:02:10] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[21:02:10] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Headphones (WH-1000XM5)
[21:02:10] [Render thread/INFO]: Sound engine started
[21:02:10] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/blocks.png-atlas
[21:02:10] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/signs.png-atlas
[21:02:10] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[21:02:10] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[21:02:10] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[21:02:10] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[21:02:10] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[21:02:11] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[21:02:11] [Render thread/WARN]: Shader rendertype_crumbling could not find uniform named ColorModulator in the specified shader program.
[21:02:11] [Render thread/WARN]: Shader rendertype_crumbling could not find uniform named ScreenSize in the specified shader program.
[21:02:11] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[21:02:11] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[21:02:11] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[21:02:18] [Worker-Main-28/ERROR]: Exception reading C:\Users\Matthew\AppData\Roaming\.minecraft\saves\Hollow Caverns\level.dat
java.util.zip.ZipException: Not in GZIP format
	at java.util.zip.GZIPInputStream.readHeader(GZIPInputStream.java:165) ~[?:?]
	at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:79) ~[?:?]
	at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:91) ~[?:?]
	at net.minecraft.class_2507.method_40059(class_2507.java:31) ~[client-intermediary.jar:?]
	at net.minecraft.class_2507.method_40058(class_2507.java:47) ~[client-intermediary.jar:?]
	at net.minecraft.class_2507.method_40057(class_2507.java:42) ~[client-intermediary.jar:?]
	at net.minecraft.class_32.method_40035(class_32.java:309) ~[client-intermediary.jar:?]
	at net.minecraft.class_32.method_29015(class_32.java:266) ~[client-intermediary.jar:?]
	at net.minecraft.class_32.method_26998(class_32.java:211) ~[client-intermediary.jar:?]
	at net.minecraft.class_32.method_43418(class_32.java:172) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[21:02:18] [Render thread/ERROR]: Invalid icon for world desolate_gully
java.lang.IllegalStateException: Must be 64 pixels wide
	at org.apache.commons.lang3.Validate.validState(Validate.java:816) ~[commons-lang3-3.12.0.jar:?]
	at net.minecraft.class_528$class_4272.method_20175(class_528.java:544) ~[client-intermediary.jar:?]
	at net.minecraft.class_528$class_4272.<init>(class_528.java:275) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_43454(class_528.java:163) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_44678(class_528.java:124) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_25394(class_528.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_526.method_25394(class_526.java:118) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_47413(class_437.java:113) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:929) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1193) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:781) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.12.jar:?]
[21:02:18] [Render thread/ERROR]: Invalid icon for world Restless Haven
java.lang.IllegalStateException: Must be 64 pixels wide
	at org.apache.commons.lang3.Validate.validState(Validate.java:816) ~[commons-lang3-3.12.0.jar:?]
	at net.minecraft.class_528$class_4272.method_20175(class_528.java:544) ~[client-intermediary.jar:?]
	at net.minecraft.class_528$class_4272.<init>(class_528.java:275) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_43454(class_528.java:163) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_44678(class_528.java:124) ~[client-intermediary.jar:?]
	at net.minecraft.class_528.method_25394(class_528.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_526.method_25394(class_526.java:118) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_47413(class_437.java:113) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:929) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1193) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:781) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.12-1.19.3.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.12.jar:?]
[21:02:19] [Render thread/INFO]: Found new data pack fabric, loading it automatically
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:dirt as it is missing following references: minecraft:string (from file/Tallgrass)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:azalea_root_replaceable as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:azalea_grows_on as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:sculk_replaceable as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:sculk_replaceable_world_gen as it is missing following references: #minecraft:sculk_replaceable (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:moss_replaceable as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:lush_ground_replaceable as it is missing following references: #minecraft:moss_replaceable (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:enderman_holdable as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:bamboo_plantable_on as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:big_dripleaf_placeable as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:overworld_carver_replaceables as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:dead_bush_may_place_on as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Worker-Main-27/ERROR]: Couldn't load tag minecraft:nether_carver_replaceables as it is missing following references: #minecraft:dirt (from vanilla)
[21:02:21] [Render thread/INFO]: Loaded 7 recipes
[21:02:22] [Render thread/ERROR]: Failed to load function rick:roll
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Whilst parsing command on line 1: Incorrect argument for command at position 49: ... run item <--[HERE]
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
Caused by: java.lang.IllegalArgumentException: Whilst parsing command on line 1: Incorrect argument for command at position 49: ... run item <--[HERE]
	at net.minecraft.class_2158.method_9195(class_2158.java:63) ~[client-intermediary.jar:?]
	at net.minecraft.class_5349.method_29451(class_5349.java:83) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	... 6 more
[21:02:22] [Render thread/ERROR]: Couldn't load tag minecraft:tick as it is missing following references: rick:roll (from file/rick)
[21:02:22] [Render thread/INFO]: Loaded 1179 advancements
[21:02:22] [Render thread/WARN]: Not all defined tags for registry ResourceKey[minecraft:root / minecraft:block] are present in data pack: minecraft:nether_carver_replaceables, minecraft:overworld_carver_replaceables
[21:02:23] [Render thread/INFO]: Applied 0 biome modifications to 0 of 63 new biomes in 2.738 ms
[21:02:23] [Server thread/INFO]: Starting integrated minecraft server version 1.19.3
[21:02:23] [Server thread/INFO]: Generating keypair
[21:02:23] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[21:02:25] [Worker-Bootstrap-7/INFO]: 4445 Datafixer optimizations took 27796 milliseconds
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 0%
[21:02:26] [Render thread/INFO]: Preparing spawn area: 33%
[21:02:27] [Render thread/INFO]: Preparing spawn area: 55%
[21:02:27] [Render thread/INFO]: Time elapsed: 3894 ms
[21:02:27] [Server thread/INFO]: Changing view distance to 12, from 10
[21:02:27] [Server thread/INFO]: Changing simulation distance to 12, from 0
[21:02:28] [Server thread/INFO]: Owehttamy[local:E:62b1a30c] logged in with entity id 249 at (120.50015536328964, 41.316412906730434, 226.078388169927)
[21:02:28] [Server thread/INFO]: Owehttamy joined the game
[21:02:28] [Render thread/INFO]: Loaded 837 advancements
[21:02:29] [Server thread/INFO]: Saving and pausing game...
[21:02:29] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:overworld
[21:02:29] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:the_nether
[21:02:29] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:the_end
[21:02:31] [Server thread/INFO]: Saving and pausing game...
[21:02:31] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:overworld
[21:02:31] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:the_nether
[21:02:31] [Server thread/INFO]: Saving chunks for level 'ServerLevel[temp]'/minecraft:the_end
[21:02:39] [Server thread/INFO]: [Not Secure] <Owehttamy> a
[21:02:39] [Render thread/INFO]: [CHAT] <Owehttamy> a
[21:02:51] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 2240ms or 44 ticks behind```
